WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController+Strings.swift

```diff
@@ -19,6 +19,11 @@ extension NotificationsViewController {
                 value: "Navigation Bar Menu Button",
                 comment: "Accessibility label for the navigation bar menu button"
             )
+            static let allNotificationsTitle = NSLocalizedString(
+                "notificationsViewController.navigationBar.filters.allNotifications",
```

Comment:
Does it make sense to use `notificationsViewController` as key here? Wouldn't we be better of naming this without technical details, something like `notifications.header.filters.all.notifications`?